tests/unit/test_ingest_service.py

```
@@ -0,0 +1,35 @@
+from app.db.confluence import ConfluencePage
+from app.services.ingest_service import IngestService
+
+
+class FakeConfluenceClient:
+    async def fetch_recent_pages(self, hours: int, limit: int) -> list[ConfluencePage]:
```

Ruff (0.15.14)

[warning] Unused method argument: `hours`

(ARG002)